pkgs/tools/admin/trivy/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "trivy";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "aquasecurity";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0mqzs0yjlqpsn2540kgl4gn77qjg3kfls1zwgjfxy66lazic3f1h";
+  };
+
+  vendorSha256 = "1jmmx83c0qbqyppx0w8d9djz2j2i7dgibq9lb3vhdz78wc40zywy";
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X main.version=v${version}"
+  ];
+
+  meta = with lib; {
+    description = "A Simple and Comprehensive Vulnerability Scanner for Containers, Suitable for CI";
```

Comment:
No need for "camel-casing" here. And the readability is bad.

Reply:
I took the description as is from trivy's about section https://github.com/aquasecurity/trivy
I can fix it